services/src/main/scala/cromwell/pipeline/service/CromwellService.scala

```diff
@@ -0,0 +1,22 @@
+package cromwell.pipeline.service
+
+import cromwell.pipeline.utils.{CromwellConfig, HttpStatusCodes}
+import play.api.libs.json.JsObject
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class CromwellService(httpClient: HttpClient, cromwellConfig: CromwellConfig)(
+  implicit executionContext: ExecutionContext
+) {
+  def getEngineStatus(): Future[Either[Exception, Int]] = {
+    val awsUrl: String = cromwellConfig.host + cromwellConfig.enginePath + "/" + cromwellConfig.version + "/status"
```

Comment:
In Scala, we prefer string interpolation over string concatenation. E.g.
`val string = s"$a/$b/status"`
rather than
`val string = a + "/" + b + "/status"`
At a first glance, it may look cryptic and weird, but it might save from some unexpected bugs (e.g. if `cromwellConfig.host` and `cromwellConfig.enginePath` were `Int`s).
Also, I think this can be class field instead of local variable.